state-chain/pallets/cf-broadcast/src/lib.rs

```diff
@@ -930,28 +791,25 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 			Timeouts::<T, I>::append(
 				frame_system::Pallet::<T>::block_number() + T::BroadcastTimeout::get(),
-				broadcast_attempt.broadcast_attempt_id,
+				broadcast_id,
 			);
 
 			Self::deposit_event(Event::<T, I>::TransactionBroadcastRequest {
-				broadcast_attempt_id: broadcast_attempt.broadcast_attempt_id,
+				broadcast_id,
 				nominee: nominated_signer,
 				transaction_payload: broadcast_attempt.transaction_payload,
 				transaction_out_id: broadcast_attempt.transaction_out_id,
 			});
 		} else {
-			log::warn!(
```

Comment:
could we please leave this log message here, I have used it recently to debug the exact issue we're fixing now haha

Reply:
Added logging here